DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/DelegateApprovals/_unapprovedDelegateExpandable.cshtml

```diff
@@ -7,12 +7,12 @@
 

```

Comment:
This file is one of the ones that has had minor styling changes (Approve Delegates)